.github/workflows/2-setup-azure-environment.yml

```diff
@@ -77,6 +77,12 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      # Pull main
+      - name: Pull main
+        run: ./.github/script/pull-main.sh
```

Comment:
There is no `.github/script/pull-main.sh` script in the repo. Is it missing from the commit?